R/dataset.R

```diff
@@ -41,6 +41,14 @@ setDatasetName <- function (x, value) {
 setDatasetDescription <- function (x, value) {
     setTupleSlot(x, "description", value)
 }
+setDatasetStartDate <- function (x, value) {
+    stopifnot(is.character(value))
+    setTupleSlot(x, "startDate", value)
```

Comment:
No, this won't work--the property on the server is "start_date". I just asked you to change the R method names. Sorry if that wasn't clear. So everywhere other than the R method names (in setGeneric and setMethod), change your code and the test fixtures back to "start_date".